src/core/unit.c

```diff
@@ -935,12 +980,12 @@ int unit_add_exec_dependencies(Unit *u, ExecContext *c) {
                     EXEC_OUTPUT_JOURNAL, EXEC_OUTPUT_JOURNAL_AND_CONSOLE,
                     EXEC_OUTPUT_KMSG, EXEC_OUTPUT_KMSG_AND_CONSOLE,
                     EXEC_OUTPUT_SYSLOG, EXEC_OUTPUT_SYSLOG_AND_CONSOLE))
-                return 0;
+                 return 0;
```

Comment:
I probably can't review this as a whole.  This line though, I think the old indentation might have been right.  Maybe double-check this.

Reply:
true, will fix